silkrpc/core/evm_debug.cpp

```diff
@@ -80,6 +80,11 @@ void to_json(nlohmann::json& json, const DebugTrace& debug_trace) {
     }
 }
 
+void to_json(nlohmann::json& json, const DebugTraceResult& debug_trace_result) {
+    json["result"] = debug_trace_result.debug_trace;
+}
+
+
```

Comment:
This will be removed using the new approach.

Reply:
Code changes according the review comment

Reply:
ok